src/app/Library/CrudPanel/Traits/Columns.php

```diff
@@ -317,6 +317,17 @@ public function setActionsColumnPriority($number)
         return $this;
     }
 
+    /**
+     * Set actions column as dropdown
+     * in the CRUD table view.
+     */
+    public function displayActionsColumnAsDropdown(): self
```

Comment:
I don't think we should do this with a `CrudPanel` method.

Ideally in future, this setting would be part of the "TableComponent" settings. 
If we do it as a method in `CrudPanel` it would be hard coupled and to remove it in future would be a breaking change. 

I know we already have more stuff in CrudPanel that fits the same category as this one, but those are the ones that we should aim to remove, not add more stuff there. 

Let me know what you think @maurohmartinez

Reply:
Good catch Pedro!